os/se/ameba/rtl_security_api_wrapper.c

```diff
@@ -16,10 +16,7 @@
 *
 ***************************************************************************************************/
 
-#include <stdint.h>
-#include <stdio.h>
-#include <stdlib.h>
-#include <string.h>
+#include <tinyara/kmalloc.h>
```

Comment:
memcpy, memset requires <string.h>
NULL does <sys/types.h>
uint32_t does <stdint.h>
itoa does <stdlib.h>

sorry for late

Reply:
I have a question about this.
The reason i remove these header is because I received comment "stdint.h" it is not need on PR#4385.
So I removed them and it have no compile error and it working fine, should I add them back?

Reply:
He left the comment to remove including <string.h>. At that PR, memcpy, memset and other APIs which are defined in string.h are not used. That's why Jeongchan left the comment. But in this PR, you use memcpy and memset so that you should include string.h.

Reply:
You should include the headers for what you use in a file. That's not TizenRT specific rule. As you said, it could work because one of header includes that inside. But it makes strong dependency between them. <tinyara/kmalloc.h> is made to make a dependency with this file. It makes sometimes it works, but sometimes it doesn't. It's not right usages.

Reply:
Thank you for the explanation, I get the point now. Sorry for the misunderstanding.
I will include the necessary header base on usage.

Reply:
I have corrected the included header, please let me know if there it is any other problem.
Thank for your help.